cmd/rook/block/block_test.go

```diff
@@ -25,14 +25,26 @@ import (
 func TestBlockCommand(t *testing.T) {
 	assert.Equal(t, "block", Cmd.Use)
 
-	commands := []string{}
+	var expectedCommands = []string{}
```

Comment:
Yes. The test was intermittently failing because in the comparison, the values in the arrays were in different order. This was the error i was getting:
```
--- FAIL: TestBlockCommand (0.00s)
        Error Trace:    block_test.go:34
	Error:		Not equal:
			expected: []string{"create", "delete", "ls", "map", "unmap"}
			received: []string{"ls", "create", "delete", "map", "unmap"
```